dsl/src/main/antlr4/com/chiralbehaviors/CoRE/workspace/dsl/Workspace.g4

```diff
@@ -218,6 +220,18 @@ classifiedAttribute:
     (defaultValue = QuotedText)?
     (sequenceNumber = Number)?
     ;
+    
+networkAuthorization:
+    classification = qualifiedName
+    '.'
+    classifier = qualifiedName 
+    ('childRelationship: ' childRelationship = qualifiedName)?
+    ('authorizedParent: ' authorizedParent = qualifiedName)?
```

Comment:
Definitely want to reverse this.  as it stands now:

isA Thing1
Thing2 isA

Which doesn't make things obvious.